jaxrs/pom.xml

```diff
@@ -107,29 +107,6 @@
             <artifactId>kafka-cdi-extension</artifactId>
```

Comment:
Oh right, I thought this was the Kafka module. I've kept that in there for now because the @kafkaconfig annotation hasn't been removed from this module yet due to [AGPUSH-2169](https://issues.jboss.org/browse/AGPUSH-2169). I'll remove the Kafka dependencies though in the next commit